src/components/BattleField.vue

```diff
@@ -10,11 +10,11 @@ const { pokemonTeam, teamDamage } = useCurrentTeamPokemon()
 <template>
   <div class="field">
     <div class="rivalPokemon">
-      <LifeBar :pokemon="pokemonRival" />
+      <LifeBar :pokemon="pokemonRival" :demage="rivalDamage" />
```

Comment:
Correct typo

Reply:
As Manuel said, most of the typos are "demage", instead of "damage"